lib/utilities/global-npm-version.js

```diff
@@ -0,0 +1,9 @@
+// Get the global npm version. You cannot require global npm modules, including
+// global npm itself, but you can still *run* global npm.
+const spawnSync = require('child_process').spawnSync;
```

Comment:
removing `npm` entirely from `ember version` actually seems like the best coarse of action, as we can tell which version of npm it bundles by looking at github.